### ISSUE TITLE
Regarding RefsDecl

### ISSUE THREAD

**Issue by PonteIneptique** [Capitains/Toolkit#49]
Looking at the `matchPattern` : 
I think `(.+)` is incorrect because it's too wide (I don't think space are allowed in CTS specs ?). Could we go for `(\w+)` ?

**Comment by balmas** [comment]
ah yes, very good point. I think that would work. Although if we really wanted to cover all valid possibilities we should probably use the [CTS Spec](http://cite-architecture.github.io/ctsurn_spec/specification.html#syntax-of-a-cts-urn) and have a regex which also excludes the reserved chars.

**Comment by PonteIneptique** [comment]
You're absolutely right, through this look a lot like \w with UTF-8 and not only ASCII chars. I didn't think about the issue when you have Greek identifier (alpha or beta)...
